cmd/server/main.go

```diff
@@ -122,6 +122,20 @@ func main() {
 		return
 	}
 
+	if opts.logPath != "" {
+		// instead write parsable logs for logstash/splunk/etc
+		output, err := os.OpenFile(opts.logPath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			os.Stderr.WriteString(fmt.Sprintf("Cannot open log file %s: %v\n",
+				opts.logPath, err))
+			os.Exit(1)
+		}
+		defer output.Close()
+		logger.SetOutput(output)
+		logger.SetFormatter(&logrus.JSONFormatter{})
+	}
```

Comment:
Slight change (improvement) in behavior -- if you specify `-log-file ''`, the process will log to stdout. (Without these changes, it's not possible to log to stdout.)